src/include/gurt/mem.h

```diff
@@ -20,7 +20,7 @@
 extern "C" {
 #endif
 
-#include <string.h>
+#include <gurt/common.h>
```

Comment:
I guess the issue in proc is due to debug stuff getting initialized?   That's the problem with common.h, it adds too much crap on top of what is absolutely necessary.  CART-596 and DAOS-6251 touch on the issues